ABaseEntity/ABaseEntity.h

```diff
@@ -58,10 +72,6 @@ class VRBASE_API ABaseEntity : public AActor, public IBaseEntity {
     }
 
     virtual void Tick(float deltaTime) override { Super::Tick(deltaTime); }
-    virtual void PostDuplicate(EDuplicateMode::Type mode) override {
-        Super::PostDuplicate(mode);
-        IBaseEntity::PostDuplicate(mode);
-    }

```

Comment:
No need for additional `PostDuplicate` code since [we can just perform the same logic in BeginPlay and not have to worry about whether the game is a cooked build or not](https://docs.unrealengine.com/en-US/ProgrammingAndScripting/ProgrammingWithCPP/UnrealArchitecture/Actors/ActorLifecycle/index.html).